portfolio/src/main/webapp/style.css

```diff
@@ -1,23 +1,59 @@
-#content {
+#content{
   margin: 20px;
 }
 
-#nav-bar {
+#nav-bar{
   display: flex;
   flex-direction: row;
 }
 
-#scroll-buttons {
+.prim-color{
```

Comment:
[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space between the last selector and the declaration block.